website/pages/docs/plugins/sources/aws/configuration.mdx

```diff
@@ -27,6 +27,8 @@ spec:
     aws_debug: false
 ```
 
+See [tables](/docs/plugins/sources/aws/tables) for a list of all available tables.
```

Comment:
I'm adding this in a couple of places because right now our website's sidebar is so long that the page you're on is not always above the fold. We should improve this in the new docs website